.github/workflows/ci.yml

```diff
@@ -203,8 +203,27 @@ jobs:
           dd_service: 'datadog-ci'
           dd_env: 'ci'
           cpu_count: 2
-          enable_performance_statistics: true
           sca_enabled: true
+          diff_aware: true
+
+  datadog-static-analyzer-non-diff-aware:
+    runs-on: ubuntu-latest
+    name: Static Analyzer - Non Diff Aware
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Run Datadog static analyzer
+        id: datadog-static-analysis
+        uses: DataDog/datadog-static-analyzer-github-action@main
+        with:
+          dd_app_key: ${{ secrets.DATADOG_APP_KEY_MAIN_ACCOUNT }}
+          dd_api_key: ${{ secrets.DATADOG_API_KEY_MAIN_ACCOUNT }}
+          dd_service: 'datadog-ci'
+          dd_env: 'ci-no-dq'
```

Comment:
❓ Is the "q" in `ci-no-dq` a typo?

Reply:
Fixed. It's is for `ci-no-da` (`no-diff-aware`)

Reply:
Yep, I figured it was very likely to be azerty vs qwerty 😁